README.md

```diff
@@ -27,6 +27,7 @@ npm install pino-colada
 - [merry](https://github.com/yoshuawuyts/merry)
 - [garnish](https://github.com/mattdesl/garnish)
 - [@studio/log](https://github.com/javascript-studio/studio-log)
+- [pino-http](https://github.com/pinojs/pino-http)
```

Comment:
should (may) we add hapi-pino too (and also express-pino-logger, koa-pino-logger ?)